src/conversations/Conversations.ts

```diff
@@ -97,6 +97,13 @@ export default class Conversations {
     )
   }
 
+  /**
+   * Streams messages from all conversations.
+   *
+   * When a new conversation is initiated with the client's address, this function will automatically register it and add it to the list of conversations to watch.
+   * Callers should be aware that in some cases the first messages in a newly created legacy conversation may not be picked up by this method.
+   * This will be resolved in a future release where all new conversations will be `ConversationV2`
```

Comment:
There's still going to be the resubscription holes and the race at the beginning that could result in a conversation to be missed entirely. Do we want to make such a strong claim of this being fixed?

Reply:
I'd be inclined to just call this best effort or something and generally try to steer people away from this.

Reply:
I updated to couch the language even further. 

The only reason I'm hesitant to steer people away entirely is because they don't have any tools to do a better job than us trying to implement the same thing.